### PR TITLE
Updated Curity SDK version and dependencies

### PR DESCRIPTION
**Pull request by iggbom**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,42 +36,59 @@
...
             <artifactId>log4j-slf4j-impl</artifactId>
-            <version>2.15.0</version>
+            <version>2.20.0</version>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.30</version>
+            <version>2.0.3</version>
```
Comment (mtrojanowski) [inline pom.xml:56]: This one and the Curity SDK can have the `provided` scope.

File: pom.xml [hunk 2]
```diff
@@ -44,42 +36,59 @@
...
             <artifactId>identityserver.sdk</artifactId>
-            <version>6.6.0</version>
+            <version>8.0.0</version>
         </dependency>
-
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
-            <version>2.17.2</version>
+            <version>2.20.0</version>
```
Comment (mtrojanowski) [inline pom.xml]: I'm not sure if we even need the log4j dependencies. They don't seem to be used in the code. If we removed these three then we can also skip the dependency plugin, as only provided dependencies will be used.

**Review by mtrojanowski — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (iggbom) [reply pom.xml:56]: Added

File: pom.xml (hunk 2 above)
Comment (iggbom) [reply pom.xml]: Good point. Fixed all of that.

**Pull request merged**